audio_platform_info.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
```

Comment:
Explain the purpose of this file and why we need to add it. I never saw this on any port.

Reply:
hwaudio request the file to identify the soundcard.